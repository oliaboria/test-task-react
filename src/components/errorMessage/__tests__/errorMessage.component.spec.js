import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';

import ErrorMessage from '../errorMessage.component';

describe('ErrorMessage component', () => {
    it('should render error message', () => {
        const wrapper = shallow(<ErrorMessage />);
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});
