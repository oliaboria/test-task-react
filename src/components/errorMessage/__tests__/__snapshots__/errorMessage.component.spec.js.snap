// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage component should render error message 1`] = `
<h2
  className="error-header"
>
  Ops...Something went wrong. Please try again later
</h2>
`;
